app/modules/transcribe/aggregations/aggregations.factory.js

```diff
@@ -1,15 +1,28 @@
 'use strict';
 
 var _ = require('lodash');
+var GraphQLClient = require('graphql-request').GraphQLClient;
+
+var MIN_NUMBER_VIEWS = 4;
+var MIN_CONSENSUS_SCORE = 3;
 
 require('./aggregations.module.js')
     .factory('AggregationsFactory', AggregationsFactory);
 
 // @ngInject
-function AggregationsFactory($q, SubjectsFactory, zooAPI, zooAPIConfig, zooAPIProject) {
+function AggregationsFactory($q, SubjectsFactory, zooAPI, zooAPIConfig, zooAPIProject, appConfig) {
 
     var factory;
+
     var _aggregations = [];
+    var _client = new GraphQLClient(appConfig.graphqlEndpoint);
+    var _query = `query Aggregation($workflowId: ID!, $subjectId: ID!) {
+        workflow(id: $workflowId) {
+            reductions(subjectId: $subjectId) {
```

Comment:
Looks like you're only interested in the points+text clustering? In that case you should specify the reducer key you want: `reductions(subjectId: 1276468, reducerKey: "poly_line_text")`. Order of reductions is not guaranteed, so this will also help with https://github.com/zooniverse/shakespeares_world/pull/345/files#diff-b3ba4f0c9757d9c9f1c449df29447795R64 where you might sometimes get the results from the wrong reducer otherwise. (You'll need to change that to just be `[0]` or `first()`)

Reply:
thanks 👍